### DISCUSSION TITLE
TOC 적용 안됨

### DISCUSSION THREAD

**Discussion by lluke0** [pronist/hELLO#74] [Q&A] [answered]
**환경 (버그가 발생한 환경을 적어주세요.):**

- 운영체제: windows
- 브라우저: 네이버 웨일
- 버전: 3.13
- 스킨 버전 3.5.5

**설명**
toc가 적용이 안됩니다

**발생 과정**
https://github.com/pronist/hELLO/issues/33
위 링크에 첨부된 사진처럼 toc를 적용하고 싶습니다.

하지만 스킨 편집 - toc 체크를 했음에도 불구하고 toc가 보이지 않습니다 ㅠㅠ

**스크린샷 및 링크**
![toc 적용 안됨](https://user-images.githubusercontent.com/81678959/153103924-fe77e650-c919-4759-90a6-70afb4bed380.png)
![toc스킨 편집](https://user-images.githubusercontent.com/81678959/153103945-0296978f-9b7b-4d87-a6ac-1debebb994d9.png)

[해당 포스트 주소]
https://ku-hug.tistory.com/188

**Comment by pronist** [comment, marked as answer]
@koownij TOC 는 이미지 및 본문과의 충돌문제로 **해상도** 제약이 빡빡하게 적용되어 있습니다. 가로 `1820px` 를 넘어야 TOC 가 나타납니다. 저도 노트북에선 안 나오고 데스크탑에서는 나온답니다.

**Comment by lluke0** [comment]
그렇군요 답변 감사합니다!!
